inference-engine/src/mkldnn_plugin/nodes/mkldnn_concat_node.cpp

```diff
@@ -27,291 +27,304 @@ using namespace mkldnn;
 using namespace MKLDNNPlugin;
 using namespace InferenceEngine;
 
+
+bool MKLDNNConcatNode::isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept {
+    try {
+        auto concatOp = ngraph::as_type_ptr<const ngraph::op::v0::Concat>(op);
+        if (!concatOp) {
+            errorMessage = "Node is not an instance of the Concat operation.";
```

Comment:
minor: it is not fully true message. Ngraph opset can be extended with newer version of Concat operation. So you need to explicitly specify the opset: "v0::Concat". Please fix this node, MVN and transpose in one of the next PRs.